### PR TITLE
Restructure tutorials

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

File: docs/source/tutorial_external_solver.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Tutorial: Binding an external PDE solver to pyMOR
 =================================================
 
+.. include:: jupyter_init.txt
```
Comment (renefritze) [inline docs/source/tutorial_external_solver.rst:4]: Did you check if this has an effect? I'm using a bash kernel first and only switch to Python afterwards.

**Comment by renefritze** [comment]
Apparently the one azure job was in a limbo state for a while, but I managed to restart via their web ui.

File: docs/source/tutorial_external_solver.rst (hunk 1 above)
Comment (sdrave) [reply docs/source/tutorial_external_solver.rst:4]: Actually no, it does not seem to have an effect. Probably should be moved to the point where the Python kernel has been activated?

Comment (renefritze) [reply docs/source/tutorial_external_solver.rst:4]: That should do it, yes.

**Pull request merged**